R/dplyr.R

```diff
@@ -1,7 +1,7 @@
 src_monetdb <- function(dbname="demo", host = "localhost", port = 50000L, user = "monetdb",
   password = "monetdb", con = FALSE, ...) {
   if (!inherits(con, "MonetDBConnection") || !DBI::dbIsValid(con)) {
-    con <- DBI::dbConnect(MonetDB.R(), dbname = dbname , host = host, port = port,
+    con <- DBI::dbConnect(MonetDBLite(), dbname = dbname , host = host, port = port,
```

Comment:
This should also stay a call to `MonetDB.R()`. We could add another function `src_monetdblite()` for this without all the dbname crap and a dbdir instead.